specification/cdn/resource-manager/Microsoft.Cdn/2016-10-02/cdn.json

```diff
@@ -343,6 +343,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getSupportedOptimizationTypes": {
```

Comment:
@prakharsharma10 - Ok, just one thing to note here is that this will be ACL'd via a POST action. So, someone with a read access to a profile will not be able to get the supported optimization types for that profile. He will need to have access to this POST action.